client/task-list/tasks/Marketing/Plugin.scss

```diff
@@ -0,0 +1,42 @@
+.woocommerce-plugin-list__plugin {
+	display: flex;
+	padding: $gap-large;
+	border-top: 1px solid #e5e5e5;
+
+	&:last-child {
+		border-bottom: 1px solid #e5e5e5;
+	}
+
+	h4 {
+		margin-bottom: $gap-small;
+		font-weight: 600;
+		color: $gray-900;
+	}
+
+	p {
+		color: $gray-700;
+		font-weight: 400;
+		color: #3c434a;
```

Comment:
Should we be using a color variable? (Seeing as I am color blind, I'm not sure what this color is, so it is hard for me to suggest a specific variable that would be appropriate!)

/cc @elizaan36

Reply:
Isn't the color variable already `$gray-700`

Reply:
I was actually referring to the `color: #3c434a` -- I didn't notice that there were two `color` property declarations! One should definitely be removed.

And, definitely `border-bottom` above should be addressed too!

Reply:
Removed the errant color property in https://github.com/woocommerce/woocommerce-admin/pull/7419.  `border-bottom` does not actually exist- that was a conflict with the base branch.